app/styles/modules/_password-row.scss

```diff
@@ -297,3 +297,9 @@ input[type='text'] {
     background-color: #bdbdbe;
   }
 }
+
+//for the forgot password link
+a.reset-password {
```

Comment:
Hmm, this kind of works, but I think we'd like the focus ring to shorten to the normal length of the link text.

Peeking in the inspector, it looks like the `display: flex` on `.links` causes the link to expand. You should be able to override that in this specific instance by adding `display: block` to [`.input-help-forgot-pw` here in `app/styles/modules/_input-row.scss`](https://github.com/mozilla/fxa-content-server/blob/a3c110e99b6c6d08da88b6d5e6d3e7de0ad0346d/app/styles/modules/_input-row.scss#L100)

Reply:
Okay, thanks!